cluster/gce/gci/configure-helper.sh

```diff
@@ -1840,6 +1840,16 @@ function update-node-label() {
   done
 }
 
+# A helper function that sets file permissions for kube-controller-manager to
+# run as non root.
+function run-kube-controller-manager-as-non-root {
+  prepare-log-file /var/log/kube-controller-manager.log ${KUBE_CONTROLLER_MANAGER_RUNASUSER:-2017} ${KUBE_CONTROLLER_MANAGER_RUNASGROUP:-2017}
+  setfacl -m u:${KUBE_CONTROLLER_MANAGER_RUNASUSER:-2017}:r "${CA_CERT_BUNDLE_PATH}"
+  setfacl -m u:${KUBE_CONTROLLER_MANAGER_RUNASUSER:-2017}:r "${SERVICEACCOUNT_CERT_PATH}"
```

Comment:
What do you think about keeping the logic related to PKI in the `create-master-pki` func?

Reply:
I wanted to do that too. But ${CA_CERT_BUNDLE_PATH}" is actually written in create-node-pki, so this logic would be spread across create-master-pki and create-node-pki. We would also have to add some kind of is master check in create-node-pki.